lib/ares/responses/standard_response.rb

```diff
@@ -199,6 +199,15 @@ def address
           identification.address
         end
 
+        def to_hash
+          require 'pry'; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.